File: expectanim/src/main/java/com/github/florent37/expectanim/core/position/PositionAnimExpectationOriginal.java
package com.github.florent37.expectanim.core.position;

import android.view.View;

/**
 * Created by florentchampigny on 17/02/2017.
 */

public class PositionAnimExpectationOriginal extends PositionAnimExpectation {

    public PositionAnimExpectationOriginal() {
        setForTranslationX(true);
        setForTranslationY(true);
    }

    @Override
    public Float getCalculatedValueX(View viewToMove) {
        return 0f;
    }

    @Override
    public Float getCalculatedValueY(View viewToMove) {
        return 0f;
    }
}

File: expectanim/src/main/java/com/github/florent37/expectanim/core/position/PositionAnimExpectationOriginal.kt
package com.github.florent37.expectanim.core.position

import android.view.View

class PositionAnimExpectationOriginal : PositionAnimExpectation() {
    init {
        isForTranslationX = true
        isForTranslationY = true
    }

    override fun getCalculatedValueX(viewToMove: View): Float? {
        return 0f
    }

    override fun getCalculatedValueY(viewToMove: View): Float? {
        return 0f
    }
}
